src/ray/common/cgroup/README.md

```diff
@@ -0,0 +1,35 @@
+### Ray core cgroup documentation
+
+#### Physical execution mode
+
+Ray core supports a physical execution mode, which allows users to cap resource consumption for their applications.
+
+A few benefits:
+- It prevents application from eating up unlimited resource to starve other applications running on the same node;
+- It protects other processes on the node (including ray system components) to be killed due to insufficient resource (i.e. OOM);
+
+TODO(hjiang): reserve minimum resource will be supported in the future.
+
+#### Disclaimer and presumption
+
+- The feature is built upon cgroup, which only supports linux;
```

Comment:
`Only supports Linux. On other platforms, " + what's expected behavior. If you set enable physical mode, does it raise error, or print warning, or silently continue to run without cgroup?

Reply:
Added.

Reply:
Let's restructure it like this:

```
### Prerequisites

- Ray runs in a Linux machine that supports Cgroup V2.
- The cgroup2 filesystem is mounted at `/sys/fs/cgroup`
- Cgroup v2  Raylet has write permission to that directory.

When physical mode enabled, if any of the prerequisites unsatisfied, Ray logs an error and continue running.
```

Reply:
Adopted.